### PR TITLE
LGA-1822 Confirmation-page-with-call-back-reference-number

### PR DESCRIPTION
**Pull request by jogilder**
Added in steps for callback scenario. 
Uses several of the steps from cla_eligible_confirmation feature

### CODE REVIEW

**Review by said-moj — CHANGES_REQUESTED** [review]

File: behave/features/steps/cla_eligible_callback.py [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+# TODO this is the same as on eligible_contact so can remove once that has been merged
+def assert_form_input_element(callback_form, element_id, value):
+    element = callback_form.find_element_by_id(element_id)
+    assert element is not None
+    element.send_keys(value)
+    assert element.get_attribute('value') == value
+
+# TODO this is the same as on eligible_contact so can remove once that has been merged
+@given(u'I have passed the means test')
+def step_means_test_page(context):
```
[the remark this answers is not in the document]
Comment (jogilder) [reply behave/features/steps/cla_eligible_callback.py]: this has now been removed as the previous commit has been merged.

File: behave/features/steps/cla_eligible_confirmation.py [hunk 2]
```diff
@@ -4,6 +4,8 @@
 
 
 def assert_form_input_element(callback_form, element_id, value):
+    # import pdb
+    # pdb.set_trace()
```
[the remark this answers is not in the document]
Comment (jogilder) [reply behave/features/steps/cla_eligible_confirmation.py:8]: this is the only place it exists.

File: behave/features/cla_eligible_callback.feature [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  And I enter my phone number for the callback
+  And I select "Call on another day"
+  And I select an available day and time
+  # # All steps that are clicking continue written in identical format so can reuse code
+  And I click continue
+  Then I am taken to the "We will call you back" page located on "/result/confirmation"
+  And I should NOT see the text "You can now call CLA on 0345 345 4 345"
+  And I should see my reference number after the text "Your reference number is"
+  And A matching case should be created on the CHS
+  And The callback should have been created
```
[the remark this answers is not in the document]
Comment (jogilder) [reply behave/features/cla_eligible_callback.feature:20]: Ok, will do.

**Review by said-moj — APPROVED** [review]
👍 LGTM

**Pull request merged**